node/lib/util/stash_util.js

```diff
@@ -536,13 +542,18 @@ const makeShadowCommitForRepo = co.wrap(function *(repo,
  */
 exports.makeShadowCommit = co.wrap(function *(repo,
                                               message,
+                                              timestamp,
                                               includeMeta,
                                               includeUntracked) {
     assert.instanceOf(repo, NodeGit.Repository);
     assert.isString(message);
     assert.isBoolean(includeMeta);
     assert.isBoolean(includeUntracked);
 
+    if (null !== timestamp && Number.isNaN(timestamp)) {
```

Comment:
This is a contract violation, not a user error. Try:

```JavaScript
if (null !== timestamp) {
    assert.isNumber(timestamp);
}
```

Reply:
Tried this way, in this way, it did not throw out an error. That is why I did like this.